rmgpy/solver/base.pyx

```diff
@@ -181,8 +181,8 @@ cdef class ReactionSystem(DASx):
         return (self.__class__, (self.termination,))
     
      
-    cpdef initializeModel(self, list coreSpecies, list coreReactions, list edgeSpecies, list edgeReactions, list surfaceSpecies,
-                          list surfaceReactions, list pdepNetworks=None, atol=1e-16, rtol=1e-8, sensitivity=False, 
+    cpdef initializeModel(self, list coreSpecies, list coreReactions, list edgeSpecies, list edgeReactions, list surfaceSpecies=[],
```

Comment:
That's very interesting, I wasn't aware of that!  I haven't run into any issues with that and I expect I would have if it was an issue, but that doesn't look too difficult to remedy.